PatientWebApplication/UserMicroserviceApi/Model/Schedule.cs

```diff
@@ -0,0 +1,41 @@
+/***********************************************************************
+ * Module:  Schedule.cs
+ * Author:  Luna
+ * Purpose: Definition of the Class Employee.Schedule
+ ***********************************************************************/
+
+namespace UserMicroserviceApi.Model
+{
+    public class Schedule : Entity
+    {
+        public virtual EmployeeUser Employee { get; set; }
+        public int EmployeeId { get; set; }
+
+        public string date { get; set; }
+        public bool isOnDuty { get; set; }
+
+        public int shiftId { get; set; }
+        public virtual Shift shift { get; set; }
+
+        public string room { get; set; }
```

Comment:
Change properties names to start with upper letter and apply in rest of code (remove 'this' from constructors).